src/SFA.DAS.Commitments.Application/Commands/UpdateApprenticeship/UpdateApprenticeshipCommandHandler.cs

```diff
@@ -90,6 +92,29 @@ public sealed class UpdateApprenticeshipCommandHandler : AsyncRequestHandler<Upd
             }
         }
 
+        private async Task ResetCommitmentTransferRejection(Commitment commitment)
```

Comment:
Another choice would be to put the if condition in the command handler. (then the function reads correctly